app/containers/SearchTrackContainer/tests/reducer.test.js

```diff
@@ -0,0 +1,59 @@
+import { searchTrackContainerReducer, initialState, searchTrackContainerTypes } from '../reducer';
+
+describe('SearchTrackContainer reducer tests', () => {
+  it('should return the initial state by default', () => {
+    expect(searchTrackContainerReducer(undefined, {})).toEqual(initialState);
+  });
+
+  // it('should return the updated state when an action of type DEFAULT is dispatched', () => {
+  //   const expectedResult = { ...initialState, somePayLoad: 'Mohammed Ali Chherawalla' };
+  //   expect(
+  //     searchTrackContainerReducer(initialState, {
+  //       type: searchTrackContainerTypes.DEFAULT_ACTION,
+  //       somePayLoad: 'Mohammed Ali Chherawalla'
+  //     })
+  //   ).toEqual(expectedResult);
+  // });
+
+  it('should return the initial state when an action of type Requesting to fetch the tracks data', () => {
+    const data = { trackName: 'sol' };
+    const expectedResult = { ...initialState, tracksData: {}, searchedTrackName: undefined };
```

Comment:
Why is the searchedTrackName undefined when in actual reducer it is equal to that passed as the searchValue